package/MDAnalysis/analysis/density.py

```diff
@@ -961,13 +1260,15 @@ def Bfactor2RMSF(B):
 
         \rm{RMSF} = \sqrt{\frac{3 B}{8\pi^2}}
 
-    .. rubric:: References
+    References
+    ----------
 
     .. [Willis1975]  BTM Willis and AW Pryor. *Thermal vibrations in crystallography*. Cambridge Univ. Press, 1975
```

Comment:
Should this also have a `.. deprecated` docstring entry?

Reply:
sure